### PR TITLE
Arranger 2D

### PR DESCRIPTION
**Pull request by karol7531**
ability to arrange graphs on a plane

### CODE REVIEW

File: Assets/Scripts/GraphArranger.cs [hunk 1]
```diff
@@ -50,27 +51,25 @@ public void DisableArrangement()
     void Repel()
     {
       for (var i = 0; i < _graphService.Graph.nodes.Count; i++)
-      for (var j = i + 1; j < _graphService.Graph.nodes.Count; j++)
       {
-        var node1 = _graphService.Graph.nodes[i];
-        var node2 = _graphService.Graph.nodes[j];
+        for (var j = i + 1; j < _graphService.Graph.nodes.Count; j++)
```
Comment (arkadR) [inline Assets/Scripts/GraphArranger.cs:55]: Personally I would rather keep double loops without the brackets

File: Assets/Scripts/GraphArranger.cs [hunk 2]
```diff
@@ -92,15 +91,26 @@ void Attract()
...
 
-    float CalculateAttractVelocityMagnitude(float distance)
+    float CalculateAttractVelocityMagnitude(float distance) => Mathf.Pow(distance, _attractFunPower);
+
+    private void SquashTo2D()
     {
-      var rawResult = Mathf.Pow(distance, _attractFunPower);
-      return rawResult;
+      float avgY = _graphService.Graph.nodes.Average(n => n.Position.y);
+      foreach(var n in _graphService.Graph.nodes)
```
Comment (arkadR) [inline Assets/Scripts/GraphArranger.cs:104]: Use fuller names in foreach loops (node)

File: Assets/Scripts/GraphArrangerMode.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Assets.Scripts
+{
+  public enum GraphArrangerMode
+  {
+    _3D = 0,
+    _2D = 1
```
Comment (arkadR) [inline Assets/Scripts/GraphArrangerMode.cs:6]: I hate this limitation on enum names but it is what it is :/

File: Assets/Scripts/GraphArranger.cs (hunk 1 above)
Comment (karol7531) [reply Assets/Scripts/GraphArranger.cs:55]: i like brackets in this case

File: Assets/Scripts/GraphArrangerMode.cs (hunk 3 above)
Comment (karol7531) [reply Assets/Scripts/GraphArrangerMode.cs:6]: in this case it doesn't hurt so much

**Pull request merged**